fiaas_deploy_daemon/log_extras.py

```diff
@@ -34,6 +35,16 @@ def filter(self, record):
         return 1
 
 
+class ExtraErrorFilter(logging.Filter):
+    def filter(self, record):
+        extras = {}
+        for key in ("app_name", "namespace", "deployment_id"):
```

Comment:
It feels wrong to do both filtering and adding extra-values in the same filter.
It would make sense to separate the two and apply both filters, just to separate the concerns.